buildscripts/builddrivers.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/python3
+#########################################################################################
+#
+# Description:  This script helps to build drivers in a Windows environment for PHP 7+ (32-bit/64-bit)
+#
+# Requirement:
+#               python 3.x
+#               PHP SDK and PHP Source 
+#               Driver source code folder / GitHub repository
+#               Visual Studio 2015 (PHP 7.0* and 7.1*) and Visual Studio 2017 (PHP 7.2*)
+#
+# Execution: Run with command line with required options.
+# Examples: 
+#           py builddrivers.py (for interactive mode)
+#           py builddrivers.py -v=7.1.7 -a=x86 -t=ts -d=all -g=no
+#           py builddrivers.py --PHPVER=7.0.22 --ARCH=x64 --THREAD=nts --DRIVER=all --GITHUB=yes
+#
+# Output: Build the drivers using PHP SDK. When running locally, if build is unsuccessful, 
+#         the log file will be launched for examination. Otherwise, the drivers will be renamed 
+#         and copied to the designated location(s).
+#
+#############################################################################################
+
+import sys
+import shutil
+import os.path
+import argparse
+from buildtools import BuildUtil
+
+class BuildDriver(object):
+    """Build sqlsrv and/or pdo_sqlsrv drivers with PHP source with the following properties:
+    
+    Attributes:
+        util            # BuildUtil object whose constructor takes phpver, driver, arch, thread, debug 
+        repo            # GitHub repository
+        branch          # GitHub repository branch
+        download_source # download source from GitHub or not
+        remote_path     # remote destination to where the drivers will be placed (None for local builds)
+        local           # a boolean flag - whether the build is local
```

Comment:
This is why the whole local/remote terminology is confusing. Local and remote really just refers to where the source is, has nothing to do with where the driver binaries will be placed which in the case of the test lab this is actually not remote it's a net mount.
Anyway, the build script arg has been changed to destpath which avoids this confusion so this should be renamed to match